index.html

```diff
@@ -235,14 +255,25 @@ <h2>Working Demos</h2>
             including arctic views (not supported by most web map tiling systems).
           </p>
         </li>
-        <li><a href="https://geogratis.gc.ca/mapml/static/cubewerx_cgdi_proxy_geomet_directory.html">MapML versions of GeoMet WMS</a>
+        <li><a href="https://geogratis.gc.ca/mapml/static/us_pop_density.html">Vector example</a>
+          <p>
+            This is an HTML document containing map markup inline, representing
+            vector data according to the MapML &lt;feature&gt; proposal. </a>.
+          </p>
+        </li>
+
+        
+<!--        <li><a href="https://geogratis.gc.ca/mapml/static/cubewerx_cgdi_proxy_geomet_directory.html">MapML versions of GeoMet WMS</a>
```

Comment:
Instead of commenting this out, why not directly remove it?

Reply:
I was hoping that Cubewerx would consider my request to restore their services. But yes, I will delete it. We can always recover from the commit history.